packages/doc/content/components/components/button/link-web.mdx

```diff
@@ -7,6 +7,16 @@
 </Box>
 ```
 
+#### Using as an anchor
```

Comment:
Would be nice to push this as recommended usage for this component. How could we create a deprecation strategy here?

cc: @mmartinsoliv @tcK1 @dehmirandac2